### ISSUE TITLE
 [C4GT] Schema: Add tests for `MaxEncodedSchemaLimitExceeded`

### ISSUE THREAD

**Issue by vatsa287** [dhiway/cord#312]
## Description
Subtask under : https://github.com/cord-network/community/issues/7

New testcase can be tested under `cargo test -p pallet-schema` after adding it in the code.

## Goals

- [ ] Add tests for `MaxEncodedSchemaLimitExceeded` for pallet/schema

## Expected Outcome
- [ ] Test should assert for `MaxEncodedSchemaLimitExceeded` being returned properly in all the possible calls.
- [ ] If there are more than one function returning this error code, all of them should be validated either as one test case or as multiple test cases.
       
## Acceptance Criteria
NA

## Implementation Details
Look at other test cases, and add a test case for the same.
Ex: check method `check_duplicate_schema_creation` which validates error code `SchemaAlreadyAnchored`

## Mockups / Wireframes
NA

---- 

## Product Name
CORD

## Organization Name
Dhiway

## Domain
Blockchain

## Tech Skills Needed
Rust

## Mentor(s)
@amarts 

## Complexity
[Medium]

## Category
[Test]

## Sub Category
[Beginner friendly]

**Comment by shashank-iitbhu** [comment]
I don't think it is possible to create a schema exceeding `MaxEncodedSchemaLength` and pass it to the create function.
The `InputSchemaOf<T>` type is defined as a `BoundedVec<u8, <T as Config>::MaxEncodedSchemaLength>`, which means it is inherently limited to a maximum size specified by `MaxEncodedSchemaLength`.

So I don't think there is a need to write an explicit test for this. What do you say? or am i missing something?
@vatsa287

**Comment by shriyanyk** [comment]
Hi i would like to work on this issue.

**Comment by Pankajjsuthar** [comment]
Hi, I was currently working on this issue and find out that "schema size should be max of MaxEncodedSchemaLength and that we are getting as an input and we have to make tests which make sure that schema doesn't exceed the limit" am I right here or am i missing some details here. Can you please tell me more so that I can work on these issues, I think there are 10 in same way, so if I do one in good manner others will be easier for me.

**Comment by zeel991** [comment]
Hey @Pankajjsuthar , as you can see there is already a PR linked to the issue, so you can work on different issue

**Comment by VedantKhairnar** [comment]
Hello @Pankajjsuthar Are you open to working on this issue?
We will be happy to assign it to you now.
Thanks.

**Comment by Vaibhavsahu2810** [comment]
Hello @vatsa287 @VedantKhairnar i see that this issue has no activity for quite a while. If it's alright, i would like to work on this issue.So,please assign this issue to me.

**Comment by doxe-2611** [comment]
Hello @vatsa287  is this issue still open ? if it is open i would like to work on this issue

**Comment by devmuse7** [comment]
Hello @vatsa287 I would like to take this issue. I have an year of experience and come with hands on experience. 
Thank you.

**Comment by sujaltangde** [comment]
Hi @vatsa287 , I'd like to work on this issue. Could you please assign it to me? Thanks!

**Comment by Shivam-KSah** [comment]
Hii @vatsa287 ,I'd like to work on this issue.This issue matches with my tech inclination.Can you please assign it to me?Ready to work for the next couple of months to work on this issue as well as add some new functionalities
